### PR TITLE
Cleanup repo

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

**Comment by kentcdodds** [comment]
@all-contributors please add @MichaelDeBoey for code

File: .gitignore [hunk 1]
```diff
@@ -1,17 +1,9 @@
...
-.idea/
 
 # these cause more harm than good
 # when working with contributors
 package-lock.json
 yarn.lock
-
-# cypress recordings during a test run are temporary files
-/cypress/videos/
-/cypress/screenshots/
```
Comment (NicholasBoll) [inline .gitignore]: These should be ignored. If someone runs `npm test`, a video will be created. If a test fails, a screenshot will be created. These file will either have to be manually deleted or accidentally committed.

Comment (kentcdodds) [reply .gitignore]: haha, whoops. @MichaelDeBoey has been helping me clean up some repos and I think this was probably a copy/paste error and I just didn't review well enough. I'll fix this.

Comment (MichaelDeBoey) [reply .gitignore]: Sorry, my bad 😕

**Pull request merged**